documentation/wiki/CollectedTelemetry.md

```diff
@@ -0,0 +1,96 @@
+# MSBuild Telemetry
+
+MSBuild emits and collects telemetry to guide decisions on modernization and optimization investments. Currently the MSBuild collects telemetry only when run from SDK host (mostly the `dotnet build` and `dotnet msbuild` commands). For more details please refer to [the official SDK telemetry documentation](https://learn.microsoft.com/en-us/dotnet/core/tools/telemetry).
+
+Visual Studio collects some build related telemetry (TODO: link the specific info here) - but that is not leveraging any MSBuild instrumentation, but rather information about count and duration of MSBuild API invocations from the caller point of view.
```

Comment:
TODO ? :)

Reply:
Unfortunately I couldn't locate any official document that would list in detail what datapoints are being collected for VS. Hence the TODO to tri to dig for it.

I still wasn't successful - so I rephrased and mentioned official VS telemetry compliance doc